packages/debugger-extension/src/index.ts

```diff
@@ -271,60 +275,149 @@ const sources: JupyterFrontEndPlugin<IDebugger.ISources> = {
 const variables: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/debugger-extension:variables',
   autoStart: true,
-  requires: [IDebugger, ITranslator],
+  requires: [
+    IDebugger,
+    IDebuggerHandler,
+    ITranslator,
+    IRenderMimeRegistry,
+    IDebuggerSidebar
```

Comment:
Similar as the previous comment above: should `IDebuggerSidebar` be required? Looks like it is used to retrieve the name from the latest selection.

Reply:
The trouble comes from the context menu (as often...). In that case `args` cannot be set dynamically, here the sidebar is used to get access to the current selection when the command is triggered from the context menu.